src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -34,66 +35,53 @@ public Long getId() {
         return id;
     }
 
-    public void setId(Long id) {
-        this.id = id;
-    }
-
     public String getName() {
         return name;
     }
 
-    public void setName(String name) {
-        this.name = name;
-    }
-
     public String getColor() {
         return color;
     }
 
-    public void setColor(String color) {
-        this.color = color;
+    public void update(Line line) {
```

Comment:
null 체크가 필요하다면 자바에서 제공하는 기능을 쓰는 방법도 있습니다.

```suggestion
    public void update(Line line) {
        this.name = Objects.requireNonNull(line.getName(), "에러 메시지도 사용 가능");
        this.color = Objects.requireNonNull(line.getColor());
    }
```

Reply:
이 부분도 위 질문과 연관이 있는데 어떻게 생각하시는지 궁금합니다..!